apps/server/src/config.ts

```diff
@@ -9,6 +9,7 @@ const schema = z
     GOOGLE_APPLICATION_ID: z.string().min(1),
     GOOGLE_APPLICATION_SECRET: z.string().min(1),
     EMAILABLE_API_KEY: z.string().min(1),
+    SLACK_WEBHOOK_URL_PUBLIC_FEEDBACK: z.string().url(),
```

Comment:
```suggestion
    SLACK_WEBHOOK_URL_PUBLIC_FEEDBACK: z.string().url().optional(),
```
or else if dev does not have this on their local .env the app won't start. Since it does not impact the core fucntionality of the app we can make it optionanl